apps/els_lsp/test/els_completion_SUITE.erl

```diff
@@ -133,6 +133,11 @@ attributes(Config) ->
                 , kind => ?COMPLETION_ITEM_KIND_SNIPPET
                 , label => <<"-include().">>
                 }
+             , #{ insertText => <<"module(${1:}).">>
```

Comment:
Should we add a couple of testcases for this? E.g. one which requires quotes and one which doesn't.